newsboat: add page (#4672)
tldr-pages/tldr: pages/common/newsboat.md

# newsboat

> An RSS/Atom feed reader for the text console.
> More information: https://newsboat.org/.

> - First import feed urls from an OPML file

Reviewer: Hello @dancek, I don't understand. Are these installation instructions? We focus the tldr pages into examples for commands usage.

Reviewer 2: When you setup a feed reader, @navarroaxel, you generally import your list of feeds. These are stored in the standard OPML format :-)

To that end, this example is a great one to start with.

Author: Newsboat refuses to start at all until you have a list of feeds in `~/.newsboat/urls`. This is something like an installation instruction, yes, but it's also the main reason people would read the man page. I think it's useful.

> - First import feed urls from an OPML file
> `newsboat -i {{my-feeds.xml}}

Author: Right, this should be:
```suggestion
- First import feed urls from an OPML file:

`newsboat -i {{my-feeds.xml}}
```

> - Alternatively, add feeds manually
> `echo {{http://rss.cnn.com/rss/cnn_topstories.rss}} >> $HOME/.newsboat/urls`

Reviewer 2: Perhaps this will read better?
```suggestion
- Alternatively, add feeds manually:

`echo {{http://example.com/path/to/feed}} >> "${HOME}/.newsboat/urls"`
```

> - Start newsboat and refresh all feeds on startup
> `newsboat -r`

Reviewer 2: Perhaps this will read better?
```suggestion
- Start newsboat and refresh all feeds on startup:

`newsboat -r`
```

> - See keyboard shortcuts (the most relevant are visible in the status line)
> `?`

Reviewer 2: Perhaps this will read better?
```suggestion
- See keyboard shortcuts (the most relevant are visible in the status line):

`?`
```

# newsboat

> > An RSS/Atom feed reader for the text console.

Reviewer: ```suggestion
> An RSS/Atom feed reader for the text terminals.
```

Author: I guess `for text terminals` rather than `for the text terminals`, them.

The original form is from https://newsboat.org/ btw.
> Newsboat is an RSS/Atom feed reader for the text console. It’s an actively maintained fork of Newsbeuter.

> > An RSS/Atom feed reader for the text console.

Author: ```suggestion
> An RSS/Atom feed reader for text terminals.
```

> More information: <https://newsboat.org/>.

> - First import feed urls from an OPML file:

Reviewer: ```suggestion
- First import feed URLs from an OPML file:
```
